templates/Dockerfile-template.scratch

```diff
@@ -23,6 +23,7 @@ RUN     buildDeps="autoconf \
                    cmake \
                    curl \
                    coreutils \
+                   diffutils \
```

Comment:
that's unexpected, what was the error on alpine ?

Reply:
nevermind @GnaphronG found it : https://github.com/webmproject/libvpx/blob/master/configure#L173-L175